README.md

```diff
@@ -6,10 +6,12 @@
 
 The Adyen Flutter package provides you with the building blocks to create a seamless checkout experience for your Android and iOS Flutter app.
 
-You can integrate in two ways:
+You can integrate in several ways:
```

Comment:
Well, there are still 2 ways - DropIn or Standalone components 🤔
Maybe you can just list all components?